src/SDK/composer.json

```diff
@@ -3,6 +3,12 @@
     "description": "SDK for OpenTelemetry PHP.",
     "keywords": ["opentelemetry", "otel", "metrics", "tracing", "logging", "apm", "sdk"],
     "type": "library",
+    "support": {
+        "issues": "https://github.com/open-telemetry/opentelemetry-php/issues",
+        "source": "https://github.com/open-telemetry/opentelemetry-php",
+        "docs": "https://opentelemetry.io/docs/php",
+        "forum": "https://app.slack.com/client/T08PSQ7BQ/C01NFPCV44V"
```

Comment:
"forum" or "chat" ? https://getcomposer.org/doc/04-schema.md#support

Reply:
I feel like chat would be the more appropriate thing here, there really isn't an OT "forum" per se.